src/components/Form/Autocomplete.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import Downshift from 'downshift'
+import PropTypes from 'prop-types'
+
+import { ItemsContainer, Items, Inner, styles, itemToString } from './VirtualDropdown'
+import Field from './Field'
+
+const Autocomplete = ({
+  items,
+  label,
+  value,
+  onChange,
+  filter,
+  onFilterChange
+}) => {
+  return (
+    <Downshift {...{
+      onChange,
+      selectedItem: value,
+      onInputValueChange: nextFilter => onFilterChange(nextFilter || ''),
+      itemToString
+    }}>
+      {({
+        getInputProps,
+        getItemProps,
+        getLabelProps,
+        selectedItem,
+        highlightedIndex,
+        isOpen
+      }) => {
+        return <div {...styles.root}>
+          <Inner isOpen={isOpen}>
+            <Field
+              label={label}
+              renderInput={
+                ({ onBlur: onFieldBlur, ...fieldProps}) => {
+                  return <input {...{
+                    ...fieldProps,
+                    ...getInputProps({
+                      onBlur: (...args) => !!selectedItem || onFieldBlur(...args),
```

Comment:
let's make if explicit – `if (!selectedItem) { onFieldBlur(...args) }`

Reply:
but why only forward it when not selected?
<img width="717" alt="screen shot 2017-11-22 at 13 38 37" src="https://user-images.githubusercontent.com/410211/33128119-8a5f61c0-cf8b-11e7-99ff-a34d5f58a7f3.png">
there is currently a bug the first time one select something—probably a race condition where the blur goes through.

I would aim for a solution that always forwards the blur and the value (label of selectedItem) – not only the filter

Also currently one can't unselect, deleting everything in the field and blurring should trigger a `onChange` with undefined / null.

Reply:
I really can't reproduce the bug. Could you elaborate (browser, exact behavior)? 
Will respond to the suggestions a little later

Reply:
@lukasbuenger chrome 62, open https://r-styleguide-pr-41.herokuapp.com/forms#autocomplete, click field, type `J`, select `Januar`

Only happens the first time when there isn't a selected item

Reply:
also happens in safari 11 and ff 52

Reply:
Got it. Almost never use the mouse for that kind of selection. Thanks!